session/manager_http_test.go

```diff
@@ -549,6 +549,25 @@ func TestDoesSessionSatisfy(t *testing.T) {
 			} else {
 				require.NoError(t, err)
 			}
+
+			// This should still work even if the session does not have identity data attached yet...
```

Comment:
Why? :D Or rather, why was it not an issue before?

Reply:
This change makes our code more resilient against future changes with regards to expanding or not expanding the identity, for example during `whoami` calls. Previously, we would rely on the `whoami` handler to know that it needed the identity credentials later, and "pre-load" them. That's brittle. I played around with trying to enable a whoami call without expanding the identity (`GET `/sessions/whoami?expand=none`, for performance) and the handler would panic.

This change allows us to more aggressively minimize necessary queries without breaking functionality.